extra_data/write_cxi.py

```diff
@@ -429,11 +429,13 @@ def collect_data(self):
             self.image_label: VLayout(
                 shape, dtype=image_grp[self.image_label].dtype),
             'gain': VLayout(shape, dtype=image_grp['gain'].dtype),
```

Comment:
Do we need to do the same for gain, or does raw JUNGFRAU data include gain?

Reply:
Data for the gain in JUNGFRAU raw data is stored as a separate array in the same way as in proc. It is only for raw Xtdf data (at least I'm sure for AGIPD) that the gain stage is stored as an extra dimension of the data array.

I've been able to generate a vds file for the recent JUNGFRAU raw data after applying this small change.